src/components/app.tsx

```diff
@@ -212,6 +216,18 @@ export class AppComponent extends BaseComponent<IProps, IState> {
             map={ mapPath }
             isErupting={isErupting}
           />
+          { showSidebar && 
+            <MapSidebarComponent
+              width={ 500 }
```

Comment:
are we concerned about the use of magic hardcoded numbers in this way?

Reply:
when it comes to screen position data for components I don't necessarily agree with keeping them hardcoded, but I kept with the standard that was already set in the other components.

Reply:
Yeah, its true we have some hard-coded container widths at the moment.   This PR #18 includes work to dynamically resize content areas. We should get that PR merged soon. 

For now, this 500 will do in this branch, because the map has been explicitly set to that size.  We will revisit after #18 lands.

Reply:
This is magic number  comment is a good point.  Here we are setting width  and height  on the component itself.  In #18 we start to do more  dynamic layout, so I  think this is fine here for now.